core/src/main/java/org/prevayler/foundation/JournalDiskSyncStrategy.java

```diff
@@ -0,0 +1,124 @@
+package org.prevayler.foundation;
+
+import java.io.IOException;
+import java.io.SyncFailedException;
+
+/**
+ * Used to access and control FileDescriptor sync of journals.
+ *
+ * @see org.prevayler.PrevaylerFactory#configureJournalDiskSync(JournalDiskSyncStrategy)
+ * @since 2.7
+ */
+public abstract class JournalDiskSyncStrategy {
```

Comment:
I'd like to keep the introduction of this idea separate from the rest of the generic type cleanup in this pull request. Could you remove this class and the related changes, and then continue the discussion on pull request #9 if you would like to advocate for it?

Reply:
Of course, I'll do so

Reply:
Done.